src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -493,6 +507,78 @@ public int[] EndTime
             }
         }
 
+        /// <summary>
+        /// Get the list of the properties of the animation.
+        /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
In TCSACR-395, I couldn't see any "Add-ACR". so I understand that this is added as Hidden-API, then please **REMOVE** this comment. : `/// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.`
thank you.